frontend/routes/Register.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <FormLayout>
+        <Card>
+            <form>
+                <h1>Create an account</h1>
+                <TextInput id="fname" placeholder="First name" required />
+                <TextInput id="lname" placeholder="Last name" required />
+                <TextInput id="email" placeholder="Email" type="email" required />
+                <TextInput id="password" placeholder="Password" type="password" required />
+                <TextInput id="confirm" placeholder="Confirm password" type="password" required />
+                <Button class="center" content="Register" type="submit"></Button>
+            </form>
+        </Card>
+        <Card>
+            <h2>Other registration options</h2>
+            <Button content="Continue with MyMLH"></Button>
+            <Button content="Continue with Google"></Button>
```

Comment:
Can the user-facing text here (headings, placeholders, button content) be moved to locale files?

In case you need references:
- https://vue-i18n.intlify.dev/guide/
- [This commit to `he-864`](https://github.com/codeRIT/hackathon-manager/pull/868/commits/495cba28952e459eeaaab5ac0c9ed8b4cbf582ae) is a simple example of how this transfer would look in practice